projects/frontegg-app/src/lib/frontegg-app.service.ts

```diff
@@ -180,3 +179,19 @@ export class FronteggAppService {
     this.fronteggApp?.hideCheckoutDialog();
   }
 }
+
+@Injectable()
+export class FronteggLoadGuard implements CanActivate {
```

Comment:
important: pls move it to another file. I think that 2 services/classes in the same file is not a good practice.

Reply:
Moved to the same file to avoid circular dependency